openid-4-verifiable-presentations-1_0.md

```diff
@@ -283,6 +286,20 @@ The following is a non-normative example of an Authorization Request:
     &nonce=n-0S6_WzA2Mj HTTP/1.1
 ```
 
+The following is a non-normative example of a request object with a `create_request_uri``: 
+
+```
+{
+  "iss": "https://client.example.org",
+  "aud": "https://server.example.com",
+  "iat": 1541493724,
+  "exp": 1516247022,
+  "state": "af0ifjsldkj",
+  "nonce": "n-0S6_WzA2Mj",
```

Comment:
IMO, `nonce` should not be in this request, it should be returned from the `create_request_uri` endpoint.

Reply:
that's an interesting remark. Having it here ties the whole transaction to this nonce. On the other hand, this also makes the request one time use (which prevents it from being used with the URL in a QR Code printed on paper). If nonce (and state?) is returned from the create request uri, would we need some marker/state value here to allow the verifier to determine the request context?